.pnpmfile.cjs

```diff
@@ -93,6 +93,13 @@ function fixDeps( pkg ) {
 		pkg.dependencies.browserslist = '^' + pkg.dependencies.browserslist;
 	}
 
+	// Override @types/react* dependencies in order to use their specific versions
+	for ( const dep of [ '@types/react', '@types/react-dom', '@types/react-test-renderer' ] ) {
+		if ( pkg.dependencies?.[ dep ] ) {
+			pkg.dependencies[ dep ] = '17.x';
```

Comment:
Just arbitrarily forcing every dep on these packages to "17.x" seems risky to me. But I guess we'll see what happens.

I was suggesting we avoid upgrading things that would bring in a 18.x dep rather than blindly overriding them.